lib/go-tc/deliveryservices.go

```diff
@@ -78,7 +78,7 @@ type DeliveryService struct {
 }
 
 type DeliveryServiceV13 struct {
-	DeliveryServiceV12
+	DeliveryServiceV11
 	DeepCachingType   DeepCachingType `json:"deepCachingType"`
 	FQPacingRate      int             `json:"fqPacingRate,omitempty"`
```

Comment:
do you think in the nullable structs you can get rid of the omitempty's? not sure how these got in here. probably just a misunderstanding of how the api works.

Reply:
Done